packages/components/credentials/Telegram/TelegramApi.ts

```diff
@@ -0,0 +1,24 @@
+import { INodeParams, INodeCredential } from '../../src/Interface';
+
+class TelegramApi implements INodeCredential {
+    name: string;
+    version: number;
+    credentials: INodeParams[];
+
+    constructor() {
+        this.name = 'telegramApi';
+        this.version = 1.0;
+        this.credentials = [
+            {
+                label: 'Bot Token',
+                name: 'botToken',
+                type: 'string',
+                default: 'eg: 1234567890:ABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHI',
```

Comment:
would be better to leave the default as empty and have this set to `placeholder`